nextflow_schema.json

```diff
@@ -253,7 +253,45 @@
                     "type": "string",
                     "default": "None",
                     "fa_icon": "fas fa-layer-group",
-                    "help_text": "Specify the path to a local version of the ARMfinderPlus database (see the pipelines' [usage documentation](https://nf-co.re/funcscan/usage)). If no input is given, the module ARMFINDERPLUS_UPDATE will download the database for you."
+                    "help_text": "Specify the path to a local version of the ARMfinderPlus database (see the pipelines' [usage documentation](https://nf-co.re/funcscan/usage)). If no input is given, the module ARMFINDERPLUS_UPDATE will download the database for you.",
+                    "description": "Specify the path to a local version of the ARMfinderPlus database."
+                },
+                "arg_amrfinderplus_identmin": {
+                    "type": "number",
+                    "default": -1,
+                    "help_text": "Minimum percentage amino-acid identity to reference protein or nucleotide identity for nucleotide reference if a blast alignment (based on methods: BLAST or PARTIAL) was detected, otherwise NA.  <-1> means use a curated threshold if it exists and <0.9> otherwise. Setting this value to something other than <-1> will override any curated similarity cutoffs. (BLAST: alignment is > 90% of length and > 90% identity to a protein in the AMRFinderPlus database) or PARTIAL: alignment is > 50% of length, but < 90% of length and > 90% identity to the reference, and does not end at a contig boundary).\n\n> Modifies tool parameter(s):\n> - AMRFinderPlus: `--ident_min <-1>`",
+                    "description": "Minimum identity to reference sequence.",
+                    "fa_icon": "fas fa-angle-left"
+                },
+                "arg_amrfinderplus_coveragemin": {
+                    "type": "number",
+                    "default": 0.5,
+                    "description": "Minimum coverage of the reference protein.",
+                    "help_text": "Minimum proportion of reference gene covered for a BLAST-based hit analysis if a blast alignment was detected, otherwise NA. The BLAST-based hit analysis are (BLAST: alignment is > 90% of length and > 90% identity to a protein in the AMRFinderPlus database) or PARTIAL: alignment is > 50% of length, but < 90% of length and > 90% identity to the reference, and does not end at a contig boundary). For more details see [documentation](https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus#--organism-option).\n\n> Modifies tool parameter(s):\n> - AMRFinderPlus: `--coverage_min <0.5>`",
```

Comment:
```suggestion
                    "help_text": "Minimum proportion of reference gene covered for a BLAST-based hit analysis if a BLAST alignment was detected, otherwise NA. The BLAST-based hit analysis are (BLAST: alignment is > 90% of length and > 90% identity to a protein in the AMRFinderPlus database) or PARTIAL: alignment is > 50% of length, but < 90% of length and > 90% identity to the reference, and does not end at a contig boundary). For more details see [documentation](https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus#--organism-option).\n\n> Modifies tool parameter(s):\n> - AMRFinderPlus: `--coverage_min <0.5>`",
```